### ISSUE TITLE
Cannot create container with 14.11.41204.0-de image

### ISSUE THREAD

**Issue by mmenninghaus** [microsoft/navcontainerhelper#907]
```
New-NavContainer -accept_eula -containerName testde -imageName 'mcr.microsoft.com/businesscentral/onprem:14.11.41204.0-de' -doNotExportObjectsToText -TimeZoneId (Get-TimeZone).Id -shortcuts "Desktop" -alwaysPull -updateHosts -useBestContainerOS -accept_outdated -includeCSide -enableSymbolLoading -includeTestToolkit -includeTestLibrariesOnly
```

leads to:
```
NavContainerHelper is version 0.6.5.0
NavContainerHelper is running as administrator
Host is Microsoft Windows 10 Enterprise - ltsc2019
Docker Client Version is 19.03.5
Docker Server Version is 19.03.5
Pulling image mcr.microsoft.com/businesscentral/onprem:14.11.41204.0-de-ltsc2019
14.11.41204.0-de-ltsc2019: Pulling from businesscentral/onprem
Digest: sha256:e1a1b27a40b348c7207fe9df2c653867cf953a372999ad245c3cd84abac4df91
Using image mcr.microsoft.com/businesscentral/onprem:14.11.41204.0-de-ltsc2019
Creating Container testde
Version: 14.11.41204.0-de
Style: onprem
Platform: 14.0.41143.0
Generic Tag: 0.0.9.99
Container OS Version: 10.0.17763.973 (ltsc2019)
Host OS Version: 10.0.17763.1039 (ltsc2019)
A better Generic Container OS exists for your host (mcr.microsoft.com/dynamicsnav:10.0.17763.1040-generic)
Using generic image mcr.microsoft.com/dynamicsnav:10.0.17763.1040-generic
Pulling image mcr.microsoft.com/dynamicsnav:10.0.17763.1040-generic
10.0.17763.1040-generic: Pulling from dynamicsnav
Digest: sha256:6d2222b76109031d3e162575727c29783427323900eab2e0c925a02db589850b
Generic Container OS Version: 10.0.17763.1040 (ltsc2019)
Generic Tag of better generic: 0.0.9.99
WARNING: The container operating system matches the host operating system, but the revision is different.
If you encounter issues, you might want to specify -isolation hyperv
Using locale de-DE
Using process isolation
Disabling the standard eventlog dump to container log every 2 seconds (use -dumpEventLog to enable)
Files in C:\ProgramData\NavContainerHelper\Extensions\testde\my:
- AdditionalOutput.ps1
- AdditionalSetup.ps1
- MainLoop.ps1
- SetupVariables.ps1
- updatehosts.ps1
Creating container testde from image mcr.microsoft.com/dynamicsnav:10.0.17763.1040-generic
440ce15e1b2471c18ee785dd09f5daf239c4c64508acffaf6faa4fe49014eac6
Waiting for container testde to be ready
Installing Business Central
Installing Url Rewrite
Installing OpenXML
Installing DotNetCore
Starting Local SQL Server
Starting Internet Information Server
Copying Service Tier Files
Copying Web Client Files
Copying Windows Client Files
Copying PowerShell Scripts
Copying Test Assemblies
Copying TestToolKit
Copying UpgradeToolKit
Copying Extensions
Copying ClientUserSettings
Copying ReportBuilder
Changing Database Server Collation to Latin1_General_100_CI_AS
SQL Server 2017 transmits information about your installation experience, as well as other usage and performance data, to Microsoft to help improve the product. To learn more about SQL Server 2017 data processing and privacy controls, please see the Privacy Statement.
Copying Cronus database
Modifying Business Central Service Tier Config File for Docker
Creating Business Central Service Tier
Installing SIP crypto provider: 'C:\Windows\System32\NavSip.dll'
Starting Business Central Service Tier
Failed to start service 'Dynamics 365 Business Central Server [NAV] (MicrosoftDynamicsNavServer$NAV)'.
at <ScriptBlock>, C:\Run\navinstall.ps1: line 216
at <ScriptBlock>, C:\Run\start.ps1: line 114
at <ScriptBlock>, <No file>: line 1Error
Installing Business Central
Installing Url Rewrite
Installing OpenXML
Installing DotNetCore
Starting Local SQL Server
Starting Internet Information Server
Copying Service Tier Files
Copying Web Client Files
Copying Windows Client Files
Copying PowerShell Scripts
Copying Test Assemblies
Copying TestToolKit
Copying UpgradeToolKit
Copying Extensions
Copying ClientUserSettings
Copying ReportBuilder
Changing Database Server Collation to Latin1_General_100_CI_AS
SQL Server 2017 transmits information about your installation experience, as well as other usage and performance data, to Microsoft to help improve the product. To learn more about SQL Server 2017 data processing and privacy controls, please see the Privacy Statement.
Copying Cronus database
Modifying Business Central Service Tier Config File for Docker
Creating Business Central Service Tier
Installing SIP crypto provider: 'C:\Windows\System32\NavSip.dll'
Starting Business Central Service Tier
Failed to start service 'Dynamics 365 Business Central Server [NAV] (MicrosoftDynamicsNavServer$NAV)'.
at <ScriptBlock>, C:\Run\navinstall.ps1: line 216
at <ScriptBlock>, C:\Run\start.ps1: line 114
at <ScriptBlock>, <No file>: line 1
Initialization of container testde failed
In C:\Program Files\WindowsPowerShell\Modules\navcontainerhelper\0.6.5.0\ContainerHandling\Wait-NavContainerReady.ps1:44 Zeichen:17
+ ...             throw "Initialization of container $containerName failed" ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Initialization ...r testde failed:String) [], RuntimeException
    + FullyQualifiedErrorId : Initialization of container testde failed
```
Surprisingly, it works with the w1-image.
Is there maybe a problem with the new generation from generic images for de-localization?

**Comment by elweilando** [comment]
Same here with mcr.microsoft.com/businesscentral/onprem:14.10.40471.0-de

```
Starting Business Central Service Tier
Failed to start service 'Dynamics 365 Business Central Server [NAV] (MicrosoftDynamicsNavServer$NAV)'.
at <ScriptBlock>, C:\Run\navinstall.ps1: line 216
at <ScriptBlock>, C:\Run\start.ps1: line 114
at <ScriptBlock>, <No file>: line 1
Initialization of container v14onprem failed
In E:\develop\nav-docker-environment\microsoft-navcontainerhelper\ContainerHandling\Wait-NavContainerReady.ps1:44 Zeichen:17
+ ...             throw "Initialization of container $containerName failed" ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Initialization ...14onprem failed:String) [], RuntimeException
    + FullyQualifiedErrorId : Initialization of container v14onprem failed
```

**Comment by freddydk** [comment]
I can repro that - will investigate.

**Comment by freddydk** [comment]
This is an error in the generic image for BC 13.x and 14.x
I will fix this in the next round of generic image rebuild and in the meantime, it will be fixed by applying a patch in New-NavContainer in the next version (for people having the old generic image)

**Comment by freddydk** [comment]
Fixed in 0.6.5.2
